front/pages/i/_shortId/_slug.vue

```diff
@@ -322,20 +345,25 @@ export default {
           this.editMode = false
           this.updatingIdea = false
           this.$refs.notifier.success('Idea Updated!')
+          // Redirect to updated URL slug
+          if (resp.data.updateIdea.idea.slug !== this.idea.slug) {
+            this.$router.replace(
+              `/i/${this.idea.shortId}/${resp.data.updateIdea.idea.slug}`
```

Comment:
this could fail if slug will contain some non ascii chars. Use {name: '', params: } instead

Reply:
Ok, done